app.js

```diff
@@ -126,6 +126,25 @@ app.post('/product_catalog/new_product', function(req, res) {
     })
 })
 
+// Products - Remove product Route
+app.post('/product_catalog/remove_product', function (req, res) {
+
+    // Grab the necessary data from the POST request body
+    const product_name = req.body.product_name_input;
+    
+    // Form the SQL Query needed to update the product catalog
+    const rem_product_query_string = `UPDATE products SET active = 0 WHERE name = '${product_name}'` 
```

Comment:
I just thought of something. Should we use the product's `id` attribute to identify it for deletion? Since the name attribute doesn't have a `UNIQUE` constraint in the DB, I can insert two identical "Apple" products.